test/onChain.test.js

```diff
@@ -31,22 +32,23 @@ contract('Runtime', function () {
             data,
             pc: beforeState.pc,
             stepCount: 1,
-            gasRemaining: beforeState.gasRemaining,
+            gasRemaining: beforeState.gas,
             stack: beforeState.stack,
             mem: beforeState.mem,
             accounts: beforeState.accounts,
             accountsCode: beforeState.accountsCode,
             logHash: beforeState.logHash,
           }
+
         );
         // console.log('After', onChainState.stack);
 
         // 4. check that on-chain state is the same as off-chain
-        assert.deepEqual(toStr(onChainState.stack), toStr(afterState.stack), 'Stack');
-        assert.equal(onChainState.mem, afterState.mem, 'Memory');
-        assert.deepEqual(onChainState.accounts, afterState.accounts, 'Accounts');
-        assert.equal(onChainState.accountsCode, afterState.accountsCode, 'Accounts code');
-        assert.equal(onChainState.logHash, afterState.logHash, 'Log hash');
+        // assert.deepEqual(toStr(onChainState.stack), toStr(afterState.stack), 'Stack');
```

Comment:
Do they need to be commented out?

Reply:
If we are checking `hashValue` then we are also equating stack, memory so `hashValue` should be enough to compare states.